CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 3.0.0-rc.7
+### Apple AdServices support
```

Comment:
maybe we can skip the `AdServices` bit since it's not specific to this version? it was introduced in rc 5 right?

Reply:
Yeah, I was highlighting it since it's still kinda new, but it's been a while 😄

Reply:
✅